README.rst

```diff
@@ -51,6 +51,29 @@ Finally, configure the Django settings, run migrations, and launch the developme
     $ ./manage.py runserver
 
 
+Code Quality
+------------
+
+Install black <https://pypi.org/project/black/> and integrate it with your editor of choice.
+Run flake8 <http://flake8.pycqa.org/en/latest/> to ensure you don't increase the warning count or introduce errors with your commits.
+This project uses EditorConfig <https://editorconfig.org> for code consistency.
+
+Django projects should extend the standard Django settings model for project configuration. Django projects should also make use of the Django test framework for unit tests.
+
+setup.cfg contains configuration for pycodestyle, isort <https://pypi.org/project/isort/> and flake8.
+
+Unify – modifies strings to use consistent quote style
+https://pypi.org/project/unify/
+
+The virtual env directory should be named .venv and it's preferred to use Pipenv to manage the virtual environment.
```

Comment:
If we use Pipenv it can manage the virtual environment location for us outside of the project directory by default unless `PIPENV_VENV_IN_PROJECT` is set.